### PR TITLE
[#2] Set up SwiftUI project

### LINKED ISSUE #2: Set Up SwiftUI Project
## Why

Set up iOS project to speed up development cycle with familiar libraries.

## Acceptance Criteria

Set up the iOS project inside KMM module with iOS Chapter's template or with similarity as possible.

### PR DESCRIPTION
**Pull request by blyscuit**
close #2 

## What happened

Replace iOS project from Android Studio KMM template with Nimble [iOS Template](https://github.com/nimblehq/ios-templates/tree/develop).

## Insight

- Remove all UIKit related files.
- Replace UIKit with SwiftUI as `main`.
- Import `shared` KMM module as pod.
- Configure `shared` module with iOS template scheme.
- Edit some gradle settings to run iOS project from Android Studio.
- Add CI to run Danger for iOS.

## Proof Of Work

Screen Shot 2022-09-14 at 09 34 54

Screen Shot 2022-09-14 at 14 58 57

Screen Shot 2022-09-14 at 14 58 44

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,85 @@
...
+
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+
```
Comment (minhnimble) [inline .gitignore]: Looks like a redundant line break here 😁

**Review by minhnimble — APPROVED** [review]
A small comment, the rest LGTM 🚀

**Review by edgarss — APPROVED** [review]

File: iosApp/Survey/Configurations/Plists/Info.plist [hunk 2]
```diff
@@ -13,22 +13,19 @@
 	<key>CFBundleName</key>
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
-	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
+	<string>APPL</string>
```
Comment (suho) [inline iosApp/Survey/Configurations/Plists/Info.plist]: Why this change to `APPL`? 🤔

File: iosApp/bitrise.yml [hunk 3]
```diff
@@ -0,0 +1,188 @@
+---
+format_version: '11'
```
Comment (suho) [inline iosApp/bitrise.yml]: We can remove this file

File: iosApp/Podfile [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+  pod 'R.swift'
+  pod 'Resolver' # Needs Cocoapods on iOS 11 to support Resolver
+
+  # Development
+  pod 'SwiftLint'
+  pod 'Wormholy', :configurations => ['Debug Staging', 'Debug Production']
+
+  # KMM
+  platform :ios, '14.0'
   pod 'shared', :path => '../shared'
```
Comment (suho) [inline iosApp/Podfile:35]: If you want to update to
`pod 'Shared', :path => '../shared'`

You can update in `shared/build.gradle.kts` [like this](https://github.com/suho/kmm-ic/pull/86/files#diff-cf9ae3b1eb7273a914f22b1e6d22d8447f88df0e7245a5b856fb751e507b979bR23)

So we can use `import Shared` in the iOS project

File: .github/workflows/ios_review_pull_request.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        uses: actions/cache@v2
+        id: cocoapodCache
+        with:
+          path: Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
+
+      - name: Install Pods Dependencies
+        run: bundle exec pod install
```
Comment (suho) [inline .github/workflows/ios_review_pull_request.yml:75]: You did not run the test after this step

**Review by suho — CHANGES_REQUESTED** [review]

File: iosApp/Podfile (hunk 4 above)
Comment (blyscuit) [reply iosApp/Podfile:35]: This change is having issue on the CI. Will try to open a new PR for this name change specifically.

**Comment by blyscuit** [comment]
@suho I fixed your comments and added test and coverage in the workflow.

**Review by suho — APPROVED** [review]
LGTM!
